compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -96,6 +96,7 @@ pub struct ModuleConfig {
 
     pub sanitizer: SanitizerSet,
     pub sanitizer_recover: SanitizerSet,
+    pub sanitizer_dataflow_abilist: Option<Vec<String>>,
```

Comment:
Is the `Option<>` wrapper here necessary? (That is, does it encode something that is semantically different than an empty vec?)

Reply:
It doesn't do anything semantically different. The reason I used it is because there isn't a `parse_comma_list`, just a `parse_opt_comma_list`, but I could create one. What do you think?

Reply:
I think it makes sense to add a `parse_comma_list` so I went ahead and added it.

Reply:
Done.